clusterloader2/pkg/execservice/exec_service.go

```diff
@@ -82,6 +88,7 @@ func SetUpExecService(f *framework.Framework, _ config.ExecServiceConfig) error
 	mapping["Name"] = execDeploymentName
 	mapping["Namespace"] = execDeploymentNamespace
 	mapping["Replicas"] = execPodReplicas
+	mapping["ImageRegistry"] = c.ImageRegistry
```

Comment:
Test failures are relevant - there is no `c` variable here....

Reply:
ack, thanks. getting it fixed now.

Reply:
all tests are passing now.